testssl.sh

```diff
@@ -16674,7 +16699,13 @@ EOF
 }
 
 maketempf() {
-     TEMPDIR=$(mktemp -d /tmp/testssl.XXXXXX) || exit $ERR_FCREATE
+     # GH Issue #1273 - /tmp might not be writable
+     if [[ -w /tmp ]]; then
+          TEMPPATH=/tmp
+     else
+	  TEMPPATH=`pwd`
```

Comment:
please replace the backquotes https://github.com/koalaman/shellcheck/wiki/SC2006

Reply:
Awesome, I didn't know this -- thank you for the reference, too. Any thought on using the current directory? (It seemed like a ... convenient ... option?) 
```suggestion
	  TEMPPATH=$(pwd)
```

Reply:
can't tell what the best is for your scenario. It's not a pretty solution but it's probably only for your target

Reply:
As a bit of a head-scratcher, why invoke a subshell or subprocess at all?  Is `$PWD` (a variable) not available within this context?

Reply:
Thx, good point. I don't know what the content of either (``$PWD`` vs ``$(pwd)``) is when called. But supposed that works like designed ``$PWD`` should be better

Reply:
@drwetter In a terminal/shell: `mkdir sample; cd sample; pwd; echo $PWD` — `pwd` there is a program being invoked, or a "shell built-in" if you have a fancy shell.  It's the program being invoked by `$(pwd)` or the back-quoted version, which is difficult to represent in Markdown comments.

$PWD, by comparison, is not an invocation, it is not a "call", it's a [variable replacement](https://en.wikipedia.org/wiki/String_interpolation).  The value is already tracked by the shell executing the script (actually, the kernel tracks that in the process information record for each process), the variable just asks the shell to insert that value at the given location.  There are potential concerns with paths involving spaces, thus `TEMPPATH="$PWD"` use of quotes may be required.

Reply:
Please have a look at the context of the PR what Daniel is trying to achieve. Then: I wrote the code he's doing a PR for it but I can't assure the directory he's aiming to write to is writeable. That's what I meant by

> I don't know what the content of either ($PWD vs $(pwd)) is when called.